Makefile

```diff
@@ -256,6 +257,12 @@ fmt: ## Format the code
       		echo "Fixed formatting for: $(FORMATTED)"; \
       	fi
 
+.PHONY: importfmt
+importsfmt:
+	# $(GO_NOMOD) get golang.org/x/tools/cmd/goimports
+	@echo "Formatting the imports..."
+	goimports -w $(GO_DEPENDENCIES)
```

Comment:
where is goimports coming from? Don't we need a target to install it?

Reply:
It is installed in `generate.sh` for pipeline.